src/scrabble.js

```diff
@@ -1,5 +1,191 @@
-function scrabble() {
+function normalScore(aWord) {
+  if (aWord === '' || aWord === ' \t\n' || aWord === null) {
```

Comment:
Does this check for invalid characters in the argument? like: % & $ £ € = / \

Reply:
Good point. I added most of them now but backslash will need a regex probably because it keeps reading it as escaping.